src/components/Body.jsx

```diff
@@ -0,0 +1,39 @@
+import { 
+  Center,
+  Container,
+  Flex, 
+  Heading, 
+} from '@chakra-ui/react';
+import JokesCard from './JokesCard';
+import ParametersForm from './ParametersForm';
+
+export default function Body() {
+  return (
+    <>
+      <Container maxW='container.xl'>
```

Comment:
Accessibility, maybe instead of `div`(s) you can use something with semantic value, such as `main`, `section`, etc.